projects/stargazer/plugins/other/radius/radius.cpp

```diff
@@ -6,26 +6,110 @@
 
 #include <boost/asio.hpp>
 #include <string>
+#include <iterator>
 #include <iostream>
-#include <cstdint> //uint8_t, uint32_t
 
 using STG::RADIUS;
+using STG::RAD_SETTINGS;
 
 extern "C" STG::Plugin* GetPlugin()
 {
     static RADIUS plugin;
     return &plugin;
 }
 
-std::string RADIUS::GetVersion() const
+RAD_SETTINGS::RAD_SETTINGS()
+    : m_port(0)
```

Comment:
I think it should be 1812, no? And also `m_dictionaries` should be initialized to "/usr/share/freeradius/dictionary".